model/client.go

```diff
@@ -1483,6 +1483,20 @@ func (c *Client) GetPost(channelId string, postId string, etag string) (*Result,
 	}
 }
 
+func (c *Client) GetPostById(postId string, etag string) (*PostList, *ResponseMetadata) {
```

Comment:
I think you forgot the comment :) Can you add one similar to `// GetPostById returns a post and any posts in the same thread by post id